apps/web/src/components/devices/devices-tables.tsx

```diff
@@ -0,0 +1,56 @@
+'use client';
+
+import { Table, TableBody, TableCell, TableHead, TableHeader, TableRow } from "@energyleaf/ui";
+import DeviceSortButton from "./device-sort-button";
+import { SortOrder } from "@energyleaf/db/util";
+import { de } from "date-fns/locale";
+import { format } from "date-fns";
+import React from "react";
+
+interface Props {
+    devices: { name: String, created: Date | null }[];
+    sortOrder: SortOrder;
+    sortProp: String;
+
```

Comment:
Leerzeile

Reply:
Pass ich in PGE-42 an, kein Bock, das hier noch zu ändern